src/shared/services/auth/auth.js

```diff
@@ -71,30 +73,18 @@ function catchError() {
 /**
  * Gets the access token and return path, and clears the local storage.
  */
-function handleCallback() {
-  // Parse query string into object
-  const params = queryStringParser(global.location.hash);
-  if (!params) {
-    return;
-  }
-
-  // Make sure all required parameters are there
-  if (AUTH_PARAMS.some(param => params[param] === undefined)) {
-    return;
-  }
-
+function saveToken(state, accessToken) {
   // The state param must be exactly the same as the state token we
   // have saved in local storage (to prevent CSRF)
   const localStateToken = storage.getItem(STATE_TOKEN_KEY);
-  const paramStateToken = decodeURIComponent(params.state);
+  const paramStateToken = decodeURIComponent(state, accessToken);
```

Comment:
This function only accepts one parameter.

Reply:
Solved in [1308](https://github.com/Amsterdam/signals-frontend/pull/1308)